.github/workflows/third_party_licenses_check.yml

```diff
@@ -0,0 +1,50 @@
+permissions:
+  contents: read
+on:
+  push:
+# See https://docs.github.com/en/actions/using-jobs/using-concurrency
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+name: ⚖️ Third party licenses
```

Comment:
Could you describe in the name what is the worklfow doing? i was expecting one of them testing the action itself like self test with some asserts that, kind of a action integration test